test/bigtest/network/scenarios/fetch-file-extensions-success.js

```diff
@@ -1,5 +1,11 @@
-export default server => {
-  server.createList('file-extension', 3);
+export default async server => {
```

Comment:
Does it need to be async? (here and in other scenarios)

Reply:
Can get rid of that.